src/compiler/compile/render_ssr/handlers/Comment.ts

```diff
@@ -1,8 +1,10 @@
 import Renderer, { RenderOptions } from '../Renderer';
 import Comment from '../../nodes/Comment';
 
-export default function(node: Comment, renderer: Renderer, options: RenderOptions) {
-	if (options.preserveComments) {
-		renderer.append(`<!--${node.data}-->`);
-	}
+export default function(_node: Comment, _renderer: Renderer, _options: RenderOptions) {
+	// TODO preserve comments
```

Comment:
It looks like this still needs to be implemented. I guess there aren't any tests for it?

Reply:
There actually was, yeah — I was going to do comments in a follow-up PR